src/typescript/frontend/src/app/home/page.tsx

```diff
@@ -21,11 +21,25 @@ const getCachedNumMarketsFromAptosNode = unstable_cache(
   { revalidate: 10 }
 );
 
+const stringifiedFetchPriceFeedData = () =>
+  fetchPriceFeedWithMarketState({}).then((res) => stringifyJSON(res));
```

Comment:
Do you add a sort by somewhere else ? I could not find it. If not, you should sort by either price detla or daily volume.

Just realized you can't sort by price delta, since it is not a field. You might need a second migration :sweat_smile:

Also, small design note: not sure if we want to sort 100% by price delta, since that will result in a full green price feed. I think it would be more visually pleasing to have something like mostly green with some red (actually pink). Maybe sort by `(1 + price_delta) * daily_volume` ?

Reply:
Verified/updated on call